docs/book/src/clusterctl/commands/restore.md

```diff
@@ -0,0 +1,13 @@
+# clusterctl restore
```

Comment:
General comment: this command takes a set of files and applies it, however, as far as I remember, there is no mechanism ensuring the content of these files is the one generated from backup, and this raises some security concerns.
Should we add a note informing the operator about this risk/about the fact that he is responsible for ensuring that the input does not contain malicious objects?